execution_workflows/README.md

```diff
@@ -76,6 +76,32 @@ For the codes please refer to the following documents:
  `AA/MISO/AA_MISO_01.bed` would be the output of MISO (your method) for the identification challenge (OUTCODE 01, we know that from [`execution_output_specification.md`][outcode]), run on dataset "P19" using 4 cores (PARAMCODE AA, we know that from) [`summary_input_specification.md`][param-code])
 
 
+## Tools
+List of tools used in APAeval. Please update columns as the execution workflows progress.
+
+| Method | Citation | Repository | Type | Status in APAeval | Present in OpenEBench | Installation | Can take your dog for a walk? |
```

Comment:
Maybe you could replace the word `dog` with `:dog:`, `:dog2:` or `:poodle:` :dog: :dog2: :poodle: 😜

Reply:
😂  done